.github/workflows/deploy.backend-api.yml

```diff
@@ -0,0 +1,56 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and Deploy Backend API
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  AZURE_WEBAPP_PACKAGE_PATH: './app/Monitor'
+  DOTNET_VERSION: '8.x'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Setup .NET Core
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: ${{ env.DOTNET_VERSION }}
+      
+      - name: dotnet build and publish
+        run: |
+          dotnet restore
+          dotnet build --configuration Release
+          dotnet publish -c Release --property:PublishDir='${{ env.AZURE_WEBAPP_PACKAGE_PATH }}'
```

Comment:
This is missing the `-p:GitHash` msbuild property, and also the corresponding get git commit hash step. 
This isn't really documented so easy to miss - but it's in the azure pipeline, and relates to this middleware package: https://www.nuget.org/packages/UoN.AspNetCore.VersionMiddleware

Useful in exactly this kind of work - to see what version of the app is currently running.